google/cloud/pubsub/samples/pubsub_samples_common.cc

```diff
@@ -188,6 +189,23 @@ std::string CommitSchemaRevisionsForRollbackSchemaTesting(
   return first_revision_id;
 }
 
+void CleanupSchemas(google::cloud::pubsub::SchemaServiceClient& schema_admin,
+                    std::string const& project_id, absl::Time const& time_now) {
+  auto const parent = google::cloud::Project(project_id).FullName();
+  for (auto& schema : schema_admin.ListSchemas(parent)) {
+    if (!schema) continue;
+    absl::Time schema_create_time =
+        absl::FromUnixSeconds((*schema).revision_create_time().seconds()) +
+        absl::Nanoseconds((*schema).revision_create_time().nanos());
+
+    if (schema_create_time < time_now - absl::Hours(48)) {
+      google::pubsub::v1::DeleteSchemaRequest request;
+      request.set_name((*schema).name());
+      schema_admin.DeleteSchema(request);
```

Comment:
nit: we usually write `(void)schema_admin.DeleteSchema(request)` to signal that we really want to ignore the return value.